### ISSUE TITLE
expiration of the DST Root CA X3

### ISSUE THREAD

**Issue by mpricu** [VojtechMyslivec/letsencrypt-zimbra#74]
Hi,
I am using the script on Centos 7 ( certbot version is 1.11.0 and openssl version is 1.0.2k-fips) and the script cannot renew the Zimbra certificate with the following error:

"obtain-and-deploy-letsencrypt-cert.sh error verification of the issued certification failed"

I presume this is because the Letsencrypt's "DST Root CA X3" expired in September 2021 and the script should be updated.

Thank you very much for your work.

**Comment by VojtechMyslivec** [comment]
Hello.
This is a bit duplicate of #71 (I understand its a chatty discussion there). The solution is prepared in `chain` branch (#73), so please try to checkout this branch and try to renew the cert with `letsencrypt-zimbra` (future) version 0.6 

> Thank you very much for your work.

:heart:

**Comment by mpricu** [comment]
Hi,
I tried the chain branch and I've got the same error. 
For Centos 7 the correct path is /usr/bin/certbot in all the configuration files.
Thank you for all your work.

**Comment by zimico** [comment]
@mpricu I have the same result which I mention in #70 . The issue is chain.pem has 3 certs not 2 certs as expected.
Regards.

**Comment by mpricu** [comment]
@VojtechMyslivec @zimico Reading #70 comments I realized I can see the certbot_extra_args value when I was running the script, but it did not contain "--preferred-chain "ISRG Root X1" (please see the previously attached images). Maybe this is the problem.

**Comment by zimico** [comment]
In my case, I see --preferred-chain ISRG Root X1 (without double quote "ISRG Root X1"). 
Could you please share your cerbot version?

**Comment by mpricu** [comment]
@VojtechMyslivec As you can see in my second attached image, the version is 0.6. 
I downloaded the zip archive from your github for the chain branch.
I used just the options -v and -f  for my last try.

**Comment by mpricu** [comment]
Yes, you were right, the certbot version was the problem. For the Centos 7 the last rpm package for certbot is  version 1.11, which have some bugs related to the --preferred-chain parameter.
So I installed certbot version 1.20  using snap: https://certbot.eff.org/lets-encrypt/centosrhel7-nginx and I succeeded to update my certificate:

![Screenshot (126)](https://user-images.githubusercontent.com/40834170/136862067-2ba3af12-ca76-4a12-8941-619c38eb5d91.jpg)

Thank you very much for your work.

**Comment by dhcmega** [comment]
Hi, I also installed snap 1.20 and the last script for getting the certificates, but I had to rename the script from `obtain-and-deploy-letsencrypt-cert.sh` to `letsencrypt-zimbra.sh`
I also symlinked the old name to the new one, but of course it didn't work.

Every night the certbot runs and fails like this:
```
obtain-and-deploy-letsencrypt-cert.sh: warning: You are using deprecated script name, change it to 'letsencrypt-zimbra.sh'
```

I have tried to find the cron that runs it, but somehow there is no cron line.
Any ideas?
Thanks!
